modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationManager.java

```diff
@@ -84,12 +84,12 @@ public void stop() throws Exception {
     /**
      * Bootstrap configuration manager with customer user cfg.
      *
-     * @param hoconStr Customer configuration in hocon format.
+     * @param hoconPath Customer configuration in hocon format.
      * @throws InterruptedException If thread is interrupted during bootstrap.
      * @throws ExecutionException   If configuration update failed for some reason.
      */
-    public void bootstrap(@Language("HOCON") String hoconStr) throws InterruptedException, ExecutionException {
-        ConfigObject hoconCfg = ConfigFactory.parseString(hoconStr).root();
+    public void bootstrap(Path hoconPath) throws InterruptedException, ExecutionException {
```

Comment:
`configPath` maybe? Hocon is a data format. When people say JsonPath or xmlPath, they don't mean the path to the file, they mean the path inside of the file I guess.